js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/ExecuteChecksButton.tsx

```diff
@@ -0,0 +1,86 @@
+import {Button, Icon, Spinner, Tooltip} from '@dagster-io/ui-components';
+import React, {useState} from 'react';
+
+import {usePermissionsForLocation} from '../../app/Permissions';
+import {AssetKeyInput} from '../../graphql/types';
+import {useLaunchPadHooks} from '../../launchpad/LaunchpadHooksContext';
+
+export type EvaluateChecksAssetNode = {
+  assetKey: AssetKeyInput;
+  repository: {name: string; location: {name: string}};
+  jobNames: string[];
+};
+
+export const ExexcuteChecksButton = ({
```

Comment:
👍🏾

Reply:
eek - renamed in a follow-up I have in flight 🤦‍♂️